### PR TITLE
frontend mock property data structure

### PR DESCRIPTION
**Pull request by unnamedsunshine**
Updated mock data structure and rendered the page.

### CODE REVIEW

File: frontend/src/app/page.tsx [hunk 1]
```diff
@@ -15,11 +15,11 @@ function HomePage({
         category={searchParams?.category}
         search={searchParams.search}
       />
-      <LoadingCards />
-      {/* <PropertgiesContainer
-      category={searchParams?.category}
-      search={searchParams.search}
-      /> */}
+      {/* <LoadingCards /> */}
+      <PropertgiesContainer
```
Comment (winlinworks) [inline frontend/src/app/page.tsx:19]: should fix typo: PropertiesContainer

File: frontend/src/utils/actions.ts [hunk 2]
```diff
@@ -2,6 +2,54 @@
 
 import { profileSchema } from "./schemas";
 
+const MockProperty = [
```
Comment (winlinworks) [inline frontend/src/utils/actions.ts:5]: convention in many languages is to have constant names in uppercase with underscores between words. class names are usually Pascal/title case.

File: frontend/src/utils/actions.ts [hunk 3]
```diff
@@ -2,6 +2,54 @@
...
+    id: 5,
+    name: "property-5",
+    tagline: "this is property 5",
+    location: "Houston, TX",
+    image:
+      "https://a0.muscache.com/im/pictures/d1db0b4a-9830-42e2-875d-0d41820ca4e3.jpg?im_w=960",
+    price: 180,
+  },
+];
+
```
Comment (winlinworks) [inline frontend/src/utils/actions.ts:52]: on L59 below, should that be validFields or validatedFields?

**Review by winlinworks — CHANGES_REQUESTED** [review]
nice work. please see comments on typos and uppercase constants

File: frontend/src/utils/actions.ts (hunk 2 above)
Comment (unnamedsunshine) [reply frontend/src/utils/actions.ts:5]: Javascript uses camelcase for constant. I will change it. Thank you for pointing it out.

File: frontend/src/utils/actions.ts (hunk 3 above)
Comment (unnamedsunshine) [reply frontend/src/utils/actions.ts:52]: It is validatedFields. Those are related to creating user profiles, which we are not doing yet.

**Pull request merged**